src/utilities/api/types.ts

```diff
@@ -12,7 +12,7 @@ export type SquarePaymentParams = {
   item_type: string;
   quantity: number;
   seller_id: string;
-  idempotencyKey: string;
+  idempotency_key: string;
```

Comment:
this should be on the Buyer and not the line_item

Reply:
OH MY BAD, moving it now